tests/test_assessment_status.py

```diff
@@ -20,8 +20,8 @@
 from tests import TestCase, TEST_USER_ID
 
 
-def mock_qr(user_id, instrument_id, status='completed'):
-    today = datetime.utcnow()
+def mock_qr(user_id, instrument_id, status='completed', timestamp=None):
+    today = timestamp or datetime.utcnow()
```

Comment:
can we change `today` to reflect potential reality :)

how about:
`timestamp = timestamp or datetime.utcnow()`

Reply:
Isn't all time relative anyway, even the word "today"? Maybe the test is just living in the past...

:)

Yea, I'll update that